common/models/product.json

```diff
@@ -19,8 +19,7 @@
       "url": "string"
     },
     "document": {
-      "apis": 
-        {},
+      "apis": "object",
```

Comment:
I notice that this isn't being done _everywhere_ - does that mean it isn't necessary? Or that the spots that were missed are dead code?